docs/ui-ux/README.md

```diff
@@ -0,0 +1,107 @@
+
+## Goal
+
+The goal of the **User Interface** section is to provide guidance on developing web applications. Everyone should begin by reading the [General Guidance](#general-guidance) for a quick introduction to the four main aspects of every web application project. From there, readers are encouraged to dive deeper into each topic, or begin reviewing technical guidance that pertains to their engagement. All UI/UX projects should begin with a detailed design document. Review the [Design Document](#design-document) section for more details, and a template to get started.
+
+Keep in mind that like all software, there is no "right way" to build a user interface application. Leverage and trust your team's or your customer's experience and expertise for the best development experience.
+
+## General Guidance
+
+The state of web platform engineering is fast moving. There is no one-size-fits-all solution. For any team to be successful in building a UI, they need to have an understanding of the higher-level aspects of all UI project.
+
+1. [**Accessibility**](./accessibility.md) - applications must be accessible for everyone. Exclusive design may work in the immediate time frame but will create major problems in the future. Accessibility is not complicated when included from the beginning. Furthermore, at Microsoft **accessibility is never optional**.
+1. [**Usability**](./usability.md) - how effortless should it be for any given user to use the application? Do they need special training or a document to understand how to use it, or will it be intuitive?
+1. [**Maintainability**](./maintainability.md) - is the application just a proof of concept to showcase an idea for future work, or will it be an MVP and act as the starting point for a larger, production-ready application? Sometimes you don't need React or any other framework. Sometimes you need React, but not all the bells and whistles from create-react-app. Understanding project maintainability requirements can simplify an engagement’s tooling needs significantly and let folks iterate without headaches.
+1. [**Stability**](./stability.md) - what is the cost of adding a dependency? Is it actively stable/updated/maintained? If not, can you afford the tech debt (sometimes the answer can be yes!)? Could you get 90% of the way there without adding another dependency?
+
+More information is available for each general guidance section in the corresponding pages.
+
+## Design Document
+
+When beginning a UI/UX project, the first step is to produce a detailed design document that outlines the prospective solution. Design documents are meant to be a starting point, and can change throughout an engagement.
```

Comment:
It would be great to cross link into some of the "design" area document and work within them on any required revisions to the design docs area.

Reply:
Great point. I'll make note of this and will make the modifications in a follow up. Thank you!